tests/Resources/config/eventbridge.yaml

```diff
@@ -11,7 +11,7 @@ services:
         public: true
 
     my_eventbridge_consumer:
-        class: Bref\Messenger\Service\EventBridge\EventBridgeConsumer
+        class: Bref\Symfony\Messenger\Service\EventBridge\EventBridgeConsumer
```

Comment:
The namespace changed, but the tests did not fail. Are these resource files in use?